### ISSUE TITLE
Does the ground truth contain bounding box ?

### ISSUE THREAD

**Issue by Rahul-Venugopal** [jagjeet-singh/Weakly-Supervised-Object-Localization#1]
Hi ,
I am little bit confused about the ground truth. In this post the original image contains a bounding box. Does the training data contains the bounding box information or only image level labels ?

Thanks 
Rahul

**Comment by ashiqimran** [comment]
I have the same query as Rahul asked above.

**Comment by jagjeet-singh** [comment]
Just the image-level labels, hence "Weakly" supervised.

**Comment by ashiqimran** [comment]
Thanks for your response. Could you walk through how to run your code?
